server/app/assets/javascripts/phone.ts

```diff
@@ -0,0 +1,13 @@
+import Cleave from 'cleave.js'
+
+export function init() {
+  const element = document.getElementById('cf-phone-number')
+  const countryElement = document.getElementById('cf-phone-country-code')
+  if (element && countryElement) {
+    new Cleave(element, {
+      numericOnly: true,
+      blocks: [0, 3, 3, 4],
+      delimiters: ['(', ') ', '-'],
```

Comment:
**nitpick (blocking):** add a comment specifying what this config does, particularly what the `blocks` attribute does

Reply:
done.